extensions/jdbc/jdbc-h2/runtime/src/main/java/io/quarkus/jdbc/h2/runtime/graal/Engine.java

```diff
@@ -3,13 +3,19 @@
 import org.h2.engine.ConnectionInfo;
 import org.h2.engine.Session;
 
+import com.oracle.svm.core.SubstrateUtil;
 import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
 
 @TargetClass(className = "org.h2.engine.Engine")
 @Substitute
 public final class Engine {
 
+    @Substitute
+    public static org.h2.engine.Engine getInstance() {
+        return SubstrateUtil.cast(new Engine(), org.h2.engine.Engine.class);
+    }
```

Comment:
This fixes an exception that appeared with `ubi-quarkus-native-image:19.3.1-java8`:
```
Caused by: com.oracle.svm.hosted.substitute.DeletedElementException: Unsupported method org.h2.engine.Engine.getInstance() is reachable: The declaring class of this element has been substituted, but this element is not present in the substitution class
```

Reply:
This looks a lot like a GraalVM bug that was fixed a long time ago.

Anyway, let's make progress.